dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/UserSettingController.java

```diff
@@ -135,6 +135,7 @@ public String getUserSettingByKey(
             .get( key );
 
         response.setHeader( ContextUtils.HEADER_CACHE_CONTROL, CacheControl.noCache().cachePrivate().getHeaderValue() );
+        response.setHeader( "Content-Type", ContextUtils.CONTENT_TYPE_TEXT );
```

Comment:
I suggest we add a static final for the content type header just for consistency/"type safety":  `ContextUtils#HEADER_CONTENT_TYPE`. There is also `HttpHeaders` from Spring.